insight-engine/latest/support/index.md

```diff
@@ -13,4 +13,4 @@ The following are the supported platforms for Search and Insight Engine:
 | Content Services 6.2.2 | |
 | Content Services 6.2.1 | |
 
-> **Note:** Search and Insight Engine 2.0 is compatible with Java 11 as long as you run Zeppelin in a Java 8 runtime. You can do this either in a VM or separate Java 8 based server. Java 11.0.9.1 and other four part Java versions are not currently compatible due to a bug in Jetty.
+> **Note:** Search and Insight Engine 2.0 is compatible with Java 11 as long as you run Zeppelin in a Java 8 runtime. You can do this either in a VM or separate Java 8 based server. Java 11.0.9.1 and other four part Java versions are not compatible for versions 2.0.2 and below due to a bug in Jetty. This issue is resolved in 2.0.3
```

Comment:
Need a full stop at the end of the last sentence. Sorry to be picky.

Also suggest we change from "This issue is resolved in 2.0.3" to "This issue is resolved in version 2.0.3."